change/@fluentui-web-components-b4360bcf-47ad-49d8-9fb5-b50bf83cf571.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Adding horizontal scroll component",
```

Comment:
```suggestion
  "comment": "only show flippers on horizontal scroll when hovered",
```

Reply:
Also weird, I ran yarn change and it didn't run through creating the change file. Now I know why.

Reply:
Fixed this.